### ISSUE TITLE
[Enhancement] Graphics Banner

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#3532]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
We need a Banner with a Graphic background.

## Describe the solution you'd like
Build and expose a recipe for a Graphics banner - composed of standard Kirby core components.
Should be exposed as a Kirby Extension (Storybook)

In the future consider building it as a Extension component.

## Are there any additional context?
**Suggested layout**
![Banner - Graphic](https://github.com/user-attachments/assets/4d3a1126-6347-451a-8892-c3d1eae2c5e4)

https://zpl.io/o01G66v

**SPEC**

https://zpl.io/9Nq1Nkp

Figma:
https://www.figma.com/design/3Cv2f8lSOvcWcNO9LXlbvL/03-Kirby-Candidates?m=auto&node-id=2523-783&t=JYVwQm8qrmTPR0mW-1

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
# Tech Refinement
With @Fuzzy3 & @RasmusKjeldgaard

We recommend adding this as a separate component in extensions, that duplicates some of (most of) the component code of image banner. Even if we get some code duplication, there are separate design rules (such as always showing the action button), markup differences and naming differences that merit adding this as a completely separate component rather than a variant of image banner.

## Default assets
The graphic should be a set of default assets that are loaded per project like our icons/spot illustrations, with an option to override the asset for custom needs.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
